internal/cmd/get/tables/fulfillment.v1.VirtualMachine.yaml

```diff
@@ -25,3 +25,7 @@ columns:
 - header: STATE
   value: this.status.state
   type: fulfillment.v1.VirtualMachineState
+
+- header: EXTERNAL-IP
+  value: this.status.ip_address
+  type: fulfillment.v1.VirtualMachineState
```

Comment:
The `type` is used/needed only for the enumerated types, because CEL treats them as integers. Secifying the type helps the rendered know what is the actual enum type, and then it can present it nicely. In this case, as  `ip_address`  is a string that isn't needed.